os/kernel/init/os_start.c

```diff
@@ -194,31 +194,23 @@ volatile dq_queue_t g_waitingforsemaphore;
 
 /* This is the list of all tasks that are blocked waiting for a signal */
 
-#ifndef CONFIG_DISABLE_SIGNALS
 volatile dq_queue_t g_waitingforsignal;
-#endif
 
 /* This is the list of all tasks that are blocked waiting for a message
  * queue to become non-empty.
  */
 
-#ifndef CONFIG_DISABLE_MQUEUE
 volatile dq_queue_t g_waitingformqnotempty;
-#endif
 
 /* This is the list of all tasks that are blocked waiting for a message
  * queue to become non-full.
  */
 
-#ifndef CONFIG_DISABLE_MQUEUE
 volatile dq_queue_t g_waitingformqnotfull;
-#endif
 
 /* This is the list of all tasks that are blocking waiting for a page fill */
 
-#ifdef CONFIG_PAGING
 volatile dq_queue_t g_waitingforfill;
-#endif
```

Comment:
Let us not modfiy the conditions for any variables or actual code. Only for enums declarations, we will remove the conditionals.